https://sources.debian.org/data/main/x/x265/2.8-3/debian/patches/0003-Fix-build-error-on-on-ppc64le.patch

This patch also fixes building on armhf-linux and aarch64-linux

From: Jayashree <jayashree.c@multicorewareinc.com>
Date: Fri, 25 May 2018 10:26:05 +0530
Subject: Fix build error on on ppc64le

---
 source/common/param.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/common/param.cpp b/source/common/param.cpp
index 31bcbc7..4a6d0c8 100644
--- a/source/common/param.cpp
+++ b/source/common/param.cpp
@@ -633,7 +633,7 @@ int x265_param_parse(x265_param* p, const char* name, const char* value)
         if (bValueWasNull)
             p->cpuid = atobool(value);
         else
-            p->cpuid = parseCpuName(value, bError);
+            p->cpuid = parseCpuName(value, bError, false);
 #endif
     }
     OPT("fps")
